packages/backend/src/models/AnalyticsModel.ts

```diff
@@ -74,9 +77,21 @@ export class AnalyticsModel {
         chartUuid: string,
         userUuid: string,
     ): Promise<void> {
-        await this.database(AnalyticsChartViewsTableName).insert({
-            chart_uuid: chartUuid,
-            user_uuid: userUuid,
+        await this.database.transaction(async (trx) => {
+            await trx(AnalyticsChartViewsTableName).insert({
+                chart_uuid: chartUuid,
+                user_uuid: userUuid,
+            });
+            await trx(SavedChartsTableName)
+                .update({
+                    // @ts-ignore
```

Comment:
not a blocker, but what do you think about using
`// @ts-expect-error message explaining why`

Reply:
done :)